### PR TITLE
SOW-24: apply GDS fonts

### PR DESCRIPTION
**Pull request by Ro4052**
Fix issues where the GDS fonts weren't being correctly imported

### CODE REVIEW

File: scripts/cdn-post-build.cjs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "dist-cdn/assets/fonts",
+  { recursive: true }
+);
+
+// Rewrite the font paths
+const css = fs
+  .readFileSync("dist-cdn/timetable-visualisation.css")
+  .toString()
+  .replace(/url\(\/assets\/fonts/g, "url(./assets/fonts");
+fs.writeFileSync("dist-cdn/timetable-visualisation.css", css);
```
Comment (Ro4052) [inline scripts/cdn-post-build.cjs:20]: This feels like a hack but I can't think of any other approach.

We could also replace with a full URL, which might be safer, but I think this is more flexible (we'll just have to give it some thorough testing).

File: package.json [hunk 2]
```diff
@@ -9,13 +9,14 @@
     "dist"
   ],
   "scripts": {
-    "dev": "vite --open",
-    "build": "tsc --p ./tsconfig-build.json && vite build",
+    "prebuild": "node ./scripts/wrapper-pre-build.cjs",
+    "dev": "npm run prebuild && vite --open",
+    "build": "npm run prebuild && tsc --p ./tsconfig-build.json && vite build",
```
Comment (vferraro-scottlogic) [inline package.json:14]: do we need `npm run prebuild` here? isn't prebuild automatically run before build by virtue of it being a "pre-script"?

Comment (Ro4052) [reply package.json]: Good point, removed :)

File: package.json [hunk 3]
```diff
@@ -9,13 +9,14 @@
...
-    "dev": "vite --open",
+    "prebuild": "node ./scripts/wrapper-pre-build.cjs",
+    "dev": "npm run prebuild && vite --open",
     "build": "tsc --p ./tsconfig-build.json && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build:storybook": "storybook build",
-    "build:cdn": "tsc --p ./tsconfig-build.json && vite --config cdn-vite.config.ts build",
+    "build:cdn": "tsc --p ./tsconfig-build.json && vite --config cdn-vite.config.ts build && node ./scripts/cdn-post-build.cjs",
```
Comment (vferraro-scottlogic) [inline package.json:19]: this part `node ./scripts/cdn-post-build.cjs` could also be in a postbuild:cdn script

Comment (Ro4052) [reply package.json:19]: Good spot! I'll also add both of these to the README

**Review by vferraro-scottlogic — APPROVED** [review]

**Pull request merged**